functions/__fzf_preview_file.fish

```diff
@@ -1,10 +1,11 @@
 # helper function for __fzf_search_current_dir
-function __fzf_preview_file --argument-names file_path --description "Print a preview for the given file based on its file type."
+function __fzf_preview_file --description "Print a preview for the given file based on its file type."
+    set file_path $argv
```

Comment:
I can't repro the bug that this fixes. I created a dir named "dir with space" and put some files, also containing spaces in the name, and was able to preview them just fine using the code on `main`. Can you teach me how to repro the bug?

Reply:
Ah ok I see why this bug happens. It was introduced by this PR, because we have to pass the unescaped expanded token to `__fzf_preview_file`. This is a good example of additional features that introduces bugs and more complicated code...

Reply:
Sorry for the delay!

> Can you teach me how to repro the bug?

Actually I just realized there's a implicit prerequisite: you have to set `fzf_preview_dir_cmd` or the code won't run `eval $fzf_preview_dir_cmd "$file_path"` but just regular `command ls -A -F "$file_path"`.

That said, here is a video demo of the issue:

https://user-images.githubusercontent.com/44045911/111025553-d0fc9980-841f-11eb-9bb5-fd6dd646fa00.mov

> It was introduced by this PR

No it's not. We do need to pass `expanded_token` now, but that's not how the bug works.